src/pages/who-we-are/_EmployerAwards.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import { employer_awards_wrapper, image_wrapper } from './who-we-are.module.scss'
+import { employer_award_content } from './_data'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import Flex from 'features/components/atoms/flex-box'
+import LinkButton from 'features/components/atoms/link-button'
+import useRegion from 'components/hooks/use-region'
+
+const EmployerAwards = () => {
+    const { is_row, is_region_loading } = useRegion()
+    return (
+        <Flex.Box
+            direction="col"
+            container="fluid"
+            padding_block="10x"
+            md={{ padding_block: '20x' }}
+        >
+            {is_row && !is_region_loading && (
+                <div>
+                    <Typography.Heading size="medium" mb="10x" textcolor="black" align="center">
+                        <Localize translate_text={'_t_UFAwards_t_'} />
+                    </Typography.Heading>
+                    <Flex.Box
+                        justify="center"
+                        direction="col"
+                        align="center"
+                        gap="10x"
+                        mt="15x"
+                        mb="40x"
+                        md={{ direction: 'row', mt: '30x', gap: '30x' }}
+                    >
+                        <div className={image_wrapper}>
+                            <StaticImage
+                                src="../../images/common/who-we-are/uf-awards.png"
+                                loading="eager"
+                                formats={['avif', 'webp', 'auto']}
+                                alt="banner"
+                            />
```

Comment:
```suggestion
                            <StaticImage
                                src="../../images/common/who-we-are/uf-awards.png"
                                formats={['avif', 'webp', 'auto']}
                                placeholder="blurred"
                                alt="banner"
                            />
```

Reply:
if it has a fixed width and height for all devices than add `layout="fixed"` and give the **width** and **height**

Reply:
Do we need to have translation for the alternate text?